test/component/dependencyReport/app/app.js

```diff
@@ -0,0 +1,3 @@
+const { init } = require('@lumigo/opentelemetry');
```

Comment:
are you planning adding the test file? like we have in the http (httpTest.ts)

Reply:
About that, I am not sure how it works. Or whether we need a separate sets of tests, because literally what we want to see, is that no additional span is created for the dependency report, which makes the test effectively the same of the `http` folder.

Reply:
We are not sending additional spans for dependency report. We use a completely different endpoint that is not an OTLP one.